clang/test/Driver/sycl-spirv-metadata-old-model.cpp

```diff
@@ -9,7 +9,7 @@
 // RUN:  FileCheck -check-prefix CHECK-WITHOUT %s
 
 // CHECK-WITH: llvm-spirv{{.*}} "--spirv-preserve-auxdata"
-// CHECK-WITH-SAME: "-spirv-ext=-all,{{.*}},+SPV_KHR_cooperative_matrix"
```

Comment:
Do we want t remove the SPV_KHR_cooperative_matrix extension here?

Reply:
Same comment for more occurrences below..

Reply:
As I understand it - the test checks the last extension name in the string. So, since I'm adding "SPV_EXT_shader_atomic_float16_add" at the end here, I'm removing the "SPV_KHR_cooperative_matrix" extension. Isn't it so?

Reply:
This check is more about just checking the `spirv-ext` option expectations (end of option verification) and probably doesn't care about specific entries - which is what the `sycl-spirv-ext-old-model` test is for.